### PR TITLE
feat: Planetary computer resources signing

### PR DESCRIPTION
**Pull request by wikipop**

### CODE REVIEW

File: backend/schemas/landsat_api.py [hunk 1]
```diff
@@ -52,7 +53,10 @@ def _landsat_item_builder(item: Item):
                 row=item.properties["landsat:wrs_row"],
             ),
             rendered_preview=item.assets["rendered_preview"].href,
-            polygon=polygon_from_nested_list(item.geometry["coordinates"])
+            polygon=polygon_from_nested_list(item.geometry["coordinates"]),
+            green=item.assets["green"].href,
+            red=item.assets["red"].href,
+            blue=item.assets["blue"].href,
```
Comment (Norbiros) [inline backend/schemas/landsat_api.py:59]: Add all of them ig so we don't have to add more later

File: backend/settings.py [hunk 2]
```diff
@@ -6,7 +6,7 @@ class Settings(BaseSettings):
     landsat_provider_url: str = "https://planetarycomputer.microsoft.com/api/stac/v1"
     supabase_url: str = "https://<id>.supabase.co"
     supabase_key: str = "<secret key>"
-
+    nuxt_open_fetch_api_base_url: str = "http://localhost:8000"
```
Comment (Norbiros) [inline backend/settings.py:9]: Configure this pydantic object to skip unknown properties instead of providing all here

**Review by Norbiros — CHANGES_REQUESTED** [review]

File: backend/settings.py (hunk 2 above)
Comment (wikipop) [reply backend/settings.py:9]: No cos it is more verbose when MISSPELLING .ENV VARIABLES

Comment (Norbiros) [reply backend/settings.py:9]: wut

**Review by Norbiros — APPROVED** [review]

**Pull request merged**